agent/consul/peering_backend.go

```diff
@@ -51,10 +53,34 @@ func (b *PeeringBackend) GetLeaderAddress() string {
 	return b.leaderAddr
 }
 
-// GetAgentCACertificates gets the server's raw CA data from its TLS Configurator.
-func (b *PeeringBackend) GetAgentCACertificates() ([]string, error) {
-	// TODO(peering): handle empty CA pems
-	return b.srv.tlsConfigurator.GRPCManualCAPems(), nil
+// GetTLSMaterials returns the TLS materials for the dialer to dial the acceptor using TLS.
+// It returns the server name to validate, and the CA certificate to validate with.
+func (b *PeeringBackend) GetTLSMaterials(generatingToken bool) (string, []string, error) {
```

Comment:
I'm probably missing something, but am not seeing the code for falling back to the manually configured
certs. Where does that happen?

Reply:
Good catch! We decided to not have that fallback anymore. Since Connect is now required when generating tokens we're just always going to use the internally managed cert.

If someone wants to adjust details about how the CA is managed they can do that by updating the Connect CA config.

I updated the PR description to remove the fallback message.

Reply:
Awesome, thank you